package.json

```diff
@@ -36,7 +36,7 @@
     "@vue/eslint-config-typescript": "^5.0.2",
     "babel-eslint": "^10.1.0",
     "deepmerge": "^4.2.2",
-    "eslint": "^6.8.0",
+    "eslint": "^7.2.0",
```

Comment:
Failing on eslint upgrade; is Travis using wrong node version? I'd like to understand – expect another failure or two.